app/assets/stylesheets/modules/playdates.scss

```diff
@@ -0,0 +1,42 @@
+.create-playdates {
+  background: #788a6f;
```

Comment:
Color literals like `#788a6f` should only be used in variable declarations; they should be referred to via variable everywhere else.